src/main/java/com/laytonsmith/abstraction/bukkit/BukkitMCWorld.java

```diff
@@ -640,6 +640,7 @@ public CArray spawnMob(MCMobs name, String subClass, int qty, MCLocation l, Targ
 								switch(ztype) {
 									case HUSK:
 										mobType = Husk.class;
+										continue;
```

Comment:
This is incorrect.

Reply:
I know it's weird, shush! This is an example of the stupid stuff I had to for 1.11 to avoid making an even more complicated backwards compatibility system for a single function that doesn't even use the proper entity ENUM.

Reply:
I was commenting on the wrong thing. This change is perfectly fine. Look before you leap.